apps/web/styles/globals.css

```diff
@@ -34,6 +34,12 @@ select:focus {
   border-color: var(--brand-color);
 }
 
+/* ensures the increment/decrement arrows always display */
+input[type=number]::-webkit-inner-spin-button,
+input[type=number]::-webkit-outer-spin-button {
+  opacity: 1;
+}
```

Comment:
Don't know about you, but my OCT was driving me crazy when seeing number inputs with no up/down arrows shown making them bigger unnecessarily up until you hover them. This is making the up/down arrows always shown.

Reply:
I'll let @Jaibles decide on this.

Reply:
Just to be clear, as an example, we would go from this:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/467258/166952688-a1c59f65-e5e2-49f8-98b5-a629b3c3c166.png">

To this:
<img width="292" alt="image" src="https://user-images.githubusercontent.com/467258/166952774-49865cc2-681f-4ef2-a212-8c45cf8c4234.png">

Reply:
> This wouldn't have been a decision I made, likely Peer did this but I have to say I quite like not having the arrows. I imagine the primary way to interact with these will be people selecting the text & typing rather than clicking these little arrows and given how small they are now, I'm not really sure there of huge benefit to the UI so I'm ok with having them hidden until hover.

Agree with @Jaibles. The arrows seem too cramped up and I'd just type in the number than click multiple times to reach my desired value